pkgs/development/python-modules/pynacl/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, pytest, coverage, libsodium, cffi, six, hypothesis}:
+
+buildPythonPackage rec {
+  pname = "pynacl";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "pyca";
+    repo = pname;
+    rev = version;
+    sha256 = "0z9i1z4hjzmp23igyhvg131gikbrr947506lwfb3fayf0agwfv8f";
+  };
+
+  checkInputs = [ pytest coverage hypothesis ];
+  propagatedBuildInputs = [ libsodium cffi six ];
+
+  checkPhase = ''
+    coverage run --source nacl --branch -m pytest
+  '';
+  
+  meta = with stdenv.lib; {
+    maintainers = with maintainers; [ va1entin ];
+    description = "Python binding to the Networking and Cryptography (NaCl) library";
+    homepage = https://github.com/pyca/pynacl/;
+    license = licenses.asl20;
+    platforms = platforms.linux;
```

Comment:
Does the new one actually not build on MacOS?

Reply:
Actually, most python packages shouldn't specify `platforms` at all. It is set automatically.

Reply:
Sorry I forgot that, removed platforms now.